posthog/client.py

```diff
@@ -327,6 +353,11 @@ def _enqueue(self, msg):
         msg["properties"]["$lib"] = "posthog-python"
         msg["properties"]["$lib_version"] = VERSION
 
+        if geoip_disable is None:
+            geoip_disable = self.geoip_disable
+            if geoip_disable is True:
```

Comment:
```suggestion
            if geoip_disable:
```

Reply:
i thought we want to test just in case someone passes in a string of "True" or something

Reply:
The property being set is `True`, so if they make a mistake and pass in a string here, doesn't affect us, ok to accept.

But I guess the reverse argument applies to the `"False"` case. Upto you